hkmc2/shared/src/main/scala/hkmc2/semantics/Elaborator.scala

```diff
@@ -97,6 +111,10 @@ object Elaborator:
         Term.SynthSel(base.ref(Ident(base.nme)),
           new Tree.Ident(nme).withLocOf(id))(symOpt)
       def symbol = symOpt
+    final case class GetElem(val base: Elem) extends Elem:
+      def nme: Str = base.nme
+      def ref(id: Tree.Ident): Term = base.ref(id)
```

Comment:
The whole point of adding `GetElem` was to make this function return the application term. Why did you basically add a useless forwarder to a base `Elem` that doesn't do anything? Don't you think that makes no sense?